src/Render/UAExperiments.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace PhpAb\Render;
```

Comment:
'Render' is a verb, 'Renderer' is more appropriate name since it describes the component :) Nitpicking...